peksels/src/store/action-creators/photos.ts

```diff
@@ -0,0 +1,15 @@
+import { createClient } from "pexels";
+import { PEXELS_API_KEY } from "../../types/photos";
+
+export const fetchPhotos = () => {
```

Comment:
fetchPhotos must be async function, that return await client.photos.curated(params)